docs/user-guide/persistent-volumes/index.md

```diff
@@ -222,6 +222,51 @@ A volume will be in one of the following phases:
 
 The CLI will show the name of the PVC bound to the PV.
 
+### Mount Options
+
+Kubernetes administrator can specify additional mount option to be used when a Persistent Volume is
+being mounted on a node. 
+
+Mount option can be specified via an annotation on PV - `volume.beta.kubernetes.io/mount-options`. 
```

Comment:
"You can specify a mount option by using an the `volume.beta.kubernetes.io/mount-options` annotiation on your Persistent Volume."